lec06-pop-models.Rmd

```diff
@@ -2,3 +2,298 @@
+  - if your model is simple enough, it can be predictive in a broadly applicable way. We'll talk about some
+  famous ecological models, and the assignments will also include more examples. I will also include some
+  famous examples from physics, where this field is known as *nonlinear dynamics*. Many of these classic 
+  models are classic because their results weren't obvious before the model was created. 
+    - example: Lorenz model (show video)
+    - example: Lotka-volterra predator-prey model (show simulation in R)
+    - example: model from my research (show zyx vs pV)
+
+*Aside*: I do theoretical biophysics, which is quite similar to theoretical ecology in my case, 
+so all of what I'll be telling you about is techniques and concepts that I have found useful 
+or use very frequently and that I think are an important part of the theorist / data scientist toolkit. 
+
+3. In the next few lectures we will talk about models and about using R to analyze models. 
+We will first do this qualitatively - no math, then quantitatively. We will talk about fitting models to data,
+and by the end I hope that you will be able to incorporate modelling into your project, 
+whether a model you design yourself or an existing ecological model. 
+
+## Setting up a model
+
+Let's look at some examples of one-dimensional models and talk about what we can learn from these models
+qualitatively, without doing any calculations.
+
+**Dimension**: the number of variables in your model. (Not the number of parameters - 
+variables are what track the species you're interested in.)
+
+Let's write down a model for the growth of a species. We want to predict how many organisms are present 
+in the population at time $t$, and to do that we need to write down an equation that describes how the
+population changes with time. The notation for "change in population per change in time" is
+
+$$\frac{dN}{dt}$$
+
+If you've taken calculus, you will recognize this as the first derivative of $N$ with respect to $t$. 
+If your calculus is foggy in your memory, when you see this notation think "change in $N$ per change in $t$.
+$dN$ is the notation commonly used to mean "a small amount of $N$, so this expression mathematically describes
+by what small amount $N$ will change in a small amount of time $dt$. This is also sometimes written as
+$\dot{N}$. I will usually use $dN/dt$ but you may see the dot notation in the literature. 
+A single dot is shorthand for "first derivative". 
+
+Biologically, $dN/dt$ is the **growth rate** or **rate of change** of a population with $N$ organisms at time $t$. 
+
+What should the growth rate of our population depend on? Let's brainstorm some things. 
+There are no wrong answers here: anything you can think of that makes biological sense is a reasonable thing 
+to include in a model, although after brainstorming we will write something very simple as an example.
+
+Possible answers:
```

Comment:
Do you want to hide this section? Since most of the students are looking at the lecture notes as they are coding along during the lectures, so they will likely see this.